Admin/StaticContentAdmin.php

```diff
@@ -49,18 +46,13 @@ protected function configureFormFields(FormMapper $formMapper)
     {
         $formMapper
             ->with('form.group_general')
-                ->add('parentDocument', TreeModelType::class, array(
-                    'root_node' => $this->getRootPath(),
-                    'choice_list' => array(),
-                    'select_root_node' => true,
-                ))
-                ->add('name', TextType::class)
-                ->add('title', TextType::class)
-                ->add(
-                    'body',
-                    $this->ivoryCkeditor ? CKEditorType::class : TextareaType::class,
-                    $this->ivoryCkeditor
-                )
+                ->add('static_content', StaticContentType::class, [
+                    'inherit_data' => true,
```

Comment:
I put `inherit_data` in the form because it may break the form if someone wants to use it through `Symfony\Bundle\FrameworkBundle\Controller::createForm`. I'm not sure if it breaks or not, but it could be a legitimate use of the `StaticContentType`.